src/main/resources/assets/computercraft/lua/rom/apis/peripheral.lua

```diff
@@ -148,12 +161,13 @@ function wrap(name)
         return nil
     end
 
-    local result = {}
+    local result = setmetatable({}, { __name = "peripheral", name = name })
     for _, method in ipairs(methods) do
         result[method] = function(...)
             return peripheral.call(name, method, ...)
         end
     end
+    result.name = name
```

Comment:
I think it's safe to remove this line now?

Reply:
```suggestion
```

Reply:
Whoops!